update_and_build.sh

```diff
@@ -83,19 +95,14 @@ if $force_update || [[ $(git rev-list HEAD...origin/"$branch_name" --count) -gt
 
     cd "$local_dir" || exit
 
-    composer install --no-scripts
-    touch "$SCRIPT_DIR/docs/mbin-api.json"
-    php bin/console nelmio:apidoc:dump --format=json --no-pretty > "$SCRIPT_DIR/docs/mbin-api.json"
-
-    # Go back to the directory where the script is located
-    cd "$SCRIPT_DIR" || exit
+    composer install --no-scripts --no-progress
+    php bin/console nelmio:apidoc:dump --format=json --no-pretty > "$SCRIPT_DIR/docs/mbin-api.json" 2>/dev/null
```

Comment:
what it this for `2>/dev/null` ?

Reply:
The studeer are not written to console. So the run is cleaner during execution.